pkg/ui/src/views/cluster/containers/timescale/timescale.spec.tsx

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+// import { assert } from "chai";
+import { mount } from "enzyme";
+import _ from "lodash";
+import {
+  TimeScaleDropdown,
+  TimeScaleDropdownProps,
+  getTimeRangeTitle,
+  generateDisabledArrows,
+} from "./index";
+import * as timewindow from "src/redux/timewindow";
+// import sinon from "sinon";
+import moment from "moment";
+import { refreshNodes } from "src/redux/apiReducers";
+import "src/enzymeInit";
+import { MemoryRouter } from "react-router";
+import TimeFrameControls from "../../components/controls";
+import RangeSelect from "../../components/range";
+import { expect, assert } from "chai";
```

Comment:
What is a reason to use `expect` for assertions? By default `assert` is used across the entire project.